### PR TITLE
Add contract info and ABIs to /#/dapp-info

### PR DESCRIPTION
**Pull request by cuongdo**
### Checklist:

- [x] Test your work and double-check to confirm that you didn't break anything
- [x] Wrap any new text/strings for translation
- [x] Map any new environment variables with a default value in the Webpack config
- [x] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/docs)

### Description:

To aid in deployments, this change adds token & marketplace contract
info to /#/dapp-info. This also adds ABIs to help multi-sig wallets and
other wallets encode and decode contract calls.

Screenshot:

screen shot 2018-09-26 at 1 01 05 pm

### CODE REVIEW

**Comment by cuongdo** [comment]
I realize that this is quite an overloading of this page, but lack of time...

**Comment by wanderingstan** [comment]
This page is meant to be overloaded. :)

Have you ever seen the output of `info.php`?? But it's actually incredibly useful for debugging.

File: src/components/dapp-info.js [hunk 1]
```diff
@@ -1,39 +1,134 @@
...
+                <tr><th colSpan="2">Environment Variables</th></tr>
+                <tr><td>BRIDGE_SERVER_DOMAIN</td><td>{ process.env.BRIDGE_SERVER_DOMAIN }</td></tr>
+                <tr><td>BRIDGE_SERVER_PROTOCOL</td><td>{ process.env.BRIDGE_SERVER_PROTOCOL }</td></tr>
+                <tr><td>DISCOVERY_SERVER_URL</td><td>{ process.env.DISCOVERY_SERVER_URL }</td></tr>
+                <tr><td>IPFS_DOMAIN</td><td>{ process.env.IPFS_DOMAIN }</td></tr>
+                <tr><td>IPFS_GATEWAY_PORT</td><td>{ process.env.IPFS_GATEWAY_PORT }</td></tr>
+                <tr><td>IPFS_GATEWAY_PROTOCOL</td><td>{ process.env.IPFS_GATEWAY_PROTOCOL }</td></tr>
+                <tr><td>IPFS_API_PORT</td><td>{ process.env.IPFS_API_PORT }</td></tr>
+                <tr><td>IPFS_SWARM</td><td>{ process.env.IPFS_SWARM }</td></tr>
+                <tr><td>ARBITRATOR_ACCOUNT</td><td>{ process.env.ARBITRATOR_ACCOUNT }</td></tr>
```
Comment (franckc) [inline src/components/dapp-info.js]: While we are at it, do you mind adding the AFFILIATE_ACCOUNT ?

**Comment by micahalcorn** [comment]
@cuongdo if this is still open later today, I'll commit some opinionated changes. Otherwise, I'll forever hold my peace. 🙊

**Comment by wanderingstan** [comment]
I just committed some changes to make those huge abi's a little more behaved. @micahalcorn , feel free to commit some opinions too. :)

image

File: src/components/dapp-info.js (hunk 1 above)
Comment (cuongdo) [reply src/components/dapp-info.js]: done

**Comment by cuongdo** [comment]
@micahalcorn @wanderingstan please feel free to make any other opinionated changes in a separate PR. I wanted this in quickly to aid in further deploys. this was my best attempt at frontend code with the time available, so it's far from ideal

**Pull request merged**